stripe/src/main/java/com/stripe/android/paymentsheet/model/ViewState.kt

```diff
@@ -37,8 +52,23 @@ internal sealed class ViewState {
         object StartProcessing : PaymentOptions()
 
         data class FinishProcessing(
-            val onComplete: () -> Unit
-        ) : PaymentOptions()
+            val result: PaymentOptionResult
+        ) : PaymentOptions() {
+            var onComplete: () -> Unit = {}
+
+            internal companion object {
+                fun create(
+                    result: PaymentOptionResult,
```

Comment:
Does finish processing need "result" as a member variable?

Reply:
It doesn't explicitly need it, but a data class needs a constructor arg. As a followup we could probably pass this value through as an input to `ProcessResult`.

Reply:
I'll think a bit more about this. I'll abandon for now.